test/run_tests.cpp

```diff
@@ -1925,10 +1925,10 @@ static void run_test(TestCase *test_case) {
 
     if (test_case->is_parseh) {
         if (buf_len(&zig_stderr) > 0) {
-            printf("\nparseh emitted warnings:\n");
+            printf("\n!!!!! parseh emitted warnings:\n");
```

Comment:
the thing with the macro definitions is important, it needs mimic the same values as the C compiler for the os-version-arch you are compiling for. getting the wrong struct/fn/union definition could be catastrophic. i think is parsing it with no macro definitions at all.